docs/git-guide/collaborate.md

```diff
@@ -0,0 +1,40 @@
+# Collaborate
+
+## Fetching work of other Contributors
+
+At Talawa, we support collaboration a lot, as we belive in unified approaches, What happens when you would like to collaborate with another contributor and they have some work done in their own fork of Talawa? No problem! Just add their fork as a remote and Pull their changes.
```

Comment:
Change to: 
At Talawa, we love collaboration and coordination amongst contributors.  What happens when you would like to collaborate with another contributor and they have some work done in their own fork of Talawa? No problem! Just add their fork as a remote and Pull their changes.